templates/taxi/car_list.html

```diff
@@ -1,22 +1,54 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
-  <h1>
-    Car list
-    <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
-      Create
-    </a>
-  </h1>
+    <h1>
+      Car List
+      <a href="{% url 'taxi:car-create' %}" class="btn btn-primary link-to-page">
+        Create
+      </a>
+    </h1>
+{#    {% load crispy_forms_tags %}#}
```

Comment:
The line `{#    {% load crispy_forms_tags %}#}` is commented out and seems unnecessary since `crispy_forms_filters` is already loaded. Consider removing it to clean up the template.